src/components/BorrowerProfile/LendCta.vue

```diff
@@ -591,7 +561,6 @@ export default {
 
 			this.isAdding = true;
 			this.errorMsg = '';
-			this.selectedOption = Number(lendAmount) || this.selectedOption;
```

Comment:
Why was this removed? It looks like it's still being used.

Reply:
We added this to consider lendAmount different than 0, no more used, also the errorMsg.

Reply:
`selectedOption` is coming from the dropdown...

Reply:
https://github.com/kiva/ui/commit/645c6bac408a54319adb8c1613bdced17154af55#diff-09ca3cc753df5d76e7f1404c47aa4ee2f07ce9ced29b352f4739cce2c1db3168L588